test/groovy/MulticloudDeployTest.groovy

```diff
@@ -276,4 +291,63 @@ class MulticloudDeployTest extends BasePiperTest {
         assert cloudFoundryDeployRule.hasParameter('deployTool', 'cf_native')
     }
 
+    @Test
+    void cfTargetMtaExtensionDescriptorTest() {
+
+        nullScript.commonPipelineEnvironment.configuration.general.neoTargets = []
+        nullScript.commonPipelineEnvironment.configuration.general.cfTargets = [cloudFoundry1]
+
+        stepRule.step.multicloudDeploy([
+            script: nullScript
+        ])
+
+        assert cloudFoundryDeployRule.hasParameter('cloudFoundry', cloudFoundry1)
+        assert cloudFoundryDeployRule.hasParameter('mtaExtensionDescriptor', 'targetMTaDescriptor.mtaext')
+    }
+
+    @Test
+    void cfGlobalMtaExtensionDescriptorTest() {
+
+        nullScript.commonPipelineEnvironment.configuration.general.neoTargets = []
+        nullScript.commonPipelineEnvironment.configuration.general.cfTargets = [cloudFoundry2]
+
+        stepRule.step.multicloudDeploy([
+            script: nullScript
+        ])
+
+        assert cloudFoundryDeployRule.hasParameter('cloudFoundry', cloudFoundry2)
+        assert cloudFoundryDeployRule.hasParameter('mtaExtensionDescriptor', 'globalMTaDescriptor.mtaext')
+    }
+
+    @Test
+    void cfTargetParametersTest() {
+
+        nullScript.commonPipelineEnvironment.configuration.general.neoTargets = []
+        nullScript.commonPipelineEnvironment.configuration.general.cfTargets = [cloudFoundry1]
+
+        stepRule.step.multicloudDeploy([
+            script: nullScript
+        ])
+
+        assert cloudFoundryDeployRule.hasParameter('apiEndpoint', 'apiEndpoint1')
+        assert cloudFoundryDeployRule.hasParameter('appName', 'testAppName1')
+        assert cloudFoundryDeployRule.hasParameter('credentialsId', 'cfCredentialsId1')
+        assert cloudFoundryDeployRule.hasParameter('manifest', 'test.yml')
+        assert cloudFoundryDeployRule.hasParameter('mtaExtensionDescriptor', 'targetMTaDescriptor.mtaext')
+        assert cloudFoundryDeployRule.hasParameter('org', 'testOrg1')
+        assert cloudFoundryDeployRule.hasParameter('space', 'testSpace1')
+
+        assert !cloudFoundryDeployRule.hasParameter('apiEndpoint', 'apiEndpoint2')
```

Comment:
I would like to clarify: are the `!assert`s here not implied by the asserts above? What is the additional benefit? Do I miss something?

Reply:
right, done